### PR TITLE
[16.0][ADD] mgmtsystem_nonconformity_hazard: add new module

### PR DESCRIPTION
**Pull request by kaynnan**
cc @marcelsavegnago @douglascstd

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
cc @WesleyOliveira98

**Review by WesleyOliveira98 — APPROVED** [review]
LGTM

**Review by marcelsavegnago — APPROVED** [review]

**Comment by kaynnan** [comment]
ping @OCA/management-systems-maintainers

File: mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This is a bridge module between Management System and Management System
```
Comment (max3903) [inline mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst]: @kaynnan Can you be more specific?

File: mgmtsystem_nonconformity_hazard/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "version": "16.0.1.0.0",
+    "author": "Escodoo, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/management-system",
+    "license": "AGPL-3",
+    "category": "Management System",
+    "depends": ["mgmtsystem_hazard", "mgmtsystem_nonconformity"],
+    "data": [
+        "views/mgmtsystem_hazard.xml",
+        "views/mgmtsystem_nonconformity.xml",
+    ],
```
Comment (max3903) [inline mgmtsystem_nonconformity_hazard/__manifest__.py:15]: @kaynnan I would auto_install this module...

**Review by max3903 — CHANGES_REQUESTED** [review]
Few small changes

File: mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst (hunk 1 above)
Comment (kaynnan) [reply mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst]: I made changes to the description, I believe it is now more detailed

File: mgmtsystem_nonconformity_hazard/__manifest__.py (hunk 2 above)
Comment (kaynnan) [reply mgmtsystem_nonconformity_hazard/__manifest__.py:15]: auto_install included, thanks for the review

File: mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+This is a bridge module between Management System and Hazard
+
+This module contains some new features for Management System modules.
+
+Nonconformity (NC)
+
+- Add the hazard_ids field to link specific hazards.
```
Comment (max3903) [inline mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst]:
```suggestion
This module allows the user to relate hazards to a nonconformity.
```

Comment (kaynnan) [reply mgmtsystem_nonconformity_hazard/readme/DESCRIPTION.rst]: Adjustments completed, I only kept the new line mentioned

**Review by max3903 — APPROVED** [review]

**Pull request merged**